oca_dependencies.txt

```diff
@@ -1 +1 @@
-
+purchase-workflow
```

Comment:
how possibly purchase_stock_picking_return_invoicing was merged without adding this? :open_mouth: